data/json/effect_on_condition.json

```diff
@@ -1298,5 +1298,24 @@
       { "u_add_trait": "CHITIN2" },
       { "u_add_trait": "CRUSTACEAN_CARAPACE" }
     ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "dermatik_pregnancy_early",
+    "condition": { "u_has_effect": "dermatik" },
+    "effect": [
+      { "u_add_effect": "formication", "duration": "15 minutes" },
+      { "queue_eocs": "dermatik_pregnancy", "time_in_future": "1 days" }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "dermatik_pregnancy",
+    "condition": { "u_has_effect": "dermatik" },
+    "effect": [
+      { "u_message": "Your flesh crawls…  A larva tears through your flesh and begins to emerge!", "type": "bad" },
+      { "u_spawn_monster": "mon_dermatik_larva", "real_count": 1, "min_radius": 1, "max_radius": 4 },
+      { "u_lose_effect": "dermatik" }
```

Comment:
Add a new effect called dermatik_damage that ticks once and applies the damage right after this eoc fires